src/components/Inputs.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+// to create an input form to insert new guest
+class Inputs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+        id: "",
+        title: "",
+        firstName: "",
+        surname: "",
+        roomId: "",
+        checkInDate: "",
+        checkOutDate: ""
+      }
+    };
+  }
+  // creating new object of entries
+  handleChange = event => {
+    const updateduser = this.state.user;
+    updateduser[event.target.name] = event.target.value;
+    console.log(this.state.user);
+    this.setState({ user: updateduser });
+    // this.setState({
+    //   user: { ...this.state.user, [event.target.name]: event.target.value }
+    // });

```

Comment:
Keep your code clean, if some code is unused, remove it instead of commenting it.